pkg/workflows/wasm/host/module.go

```diff
@@ -100,6 +90,10 @@ type ModuleConfig struct {
 	IsUncompressed bool
```

Comment:
@MStreet3 the intent of a Labeler was to be wired alongside a logger. re: my previous comments on misuse of Labeler, I would consider a couple options:

* Remove `Emitter` and drill a `Labeler` here (Requires the updates to the Labeler API that I commented on earlier) 

Or 

* Satisfy `Emitter` through a Labeler, not through the `beholderEmitter` func. (Requires the updates to the Labeler API that I commented on earlier)